orion/database/db.go

```diff
@@ -1,66 +1,66 @@
 package database
 
 import (
-  "database/sql"
-  "fmt"
+	"database/sql"
+	"fmt"
 
-  "github.com/jmoiron/sqlx"
-  _ "github.com/go-sql-driver/mysql"
-  "github.com/golang-migrate/migrate"
-  "github.com/golang-migrate/migrate/database/mysql"
-  _ "github.com/golang-migrate/migrate/source/file"
```

Comment:
same here
The indentation should look like:
```
"github.com/...
 _ "github.com/...."
```
So the second line should be: (space) _ (space) "github.com/..."